.github/workflows/_build_artifacts.yml

```diff
@@ -200,6 +200,17 @@ jobs:
           # Exclude http-test-server from perf image builds
           - { image_prefix: perf, name: { package: http-test-server } }
 
+          - {
+              stage: release,
+              name: { package: firezone-relay },
+              arch: { platform: linux/arm/v7 },
+            }
+          - {
+              stage: release,
+              name: { package: firezone-relay },
+              arch: { platform: linux/arm64 },
+            }
```

Comment:
Not needing this is addressed in https://github.com/firezone/firezone/pull/8506.